src/main/java/org/folio/circulation/services/feefine/AccountRefundCommand.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.circulation.services.feefine;
+
+import org.folio.circulation.domain.Account;
+import org.folio.circulation.domain.AccountRefundReason;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public final class AccountRefundCommand {
```

Comment:
Similarly, I think `refund account` reads better than `account refund`

```suggestion
public final class RefundAccountCommand {
```